docs/source/developing/writing-docs.adoc

```diff
@@ -66,39 +113,48 @@ To build and serve the documentation for editing, run:
 $ make serve_docs
 ----
 
-The latter will start the https://middlemanapp.com[Middleman] server on port 4567.
-You can access the rendered documentation by browsing to http://<IP of Docker daemon>:4567.
-
-[[section-deploying-minishift-docs]]
-== Deploying Minishift documentation
-
-The Minishift documentation is deployed on _docs.openshift.org_.
-To integrate with this site we deliver a tarball containing the Minishift asciidoc files as well as some link:http://www.asciibinder.org/[AsciiBinder] meta data.
+The `make serve_docs` command builds and generates a live-preview of the documentation.
+You can access the documentation by browsing to _http://<IP-of-Docker-daemon>:4567_.
 
-This tarball can be built using:
+If you encounter issues with the local staging of the documentation, you can run the following
+command to remove all build artefacts:
 
 ----
-$ make gen_adoc_tar
+$ make clean_docs
 ----
 
-Once the build completes there will be a _minishift-adoc.tar_ in the _docs/build_ directory
-of your checkout.
+[[auto-deploy-to-docs-openshift-org]]
+== Automatic deployment to docs.openshift.org
```

Comment:
Can we be a bit more neutral here. Deployment is not the key here, nor is it automatic. I suggest:

"Integration with docs.openshift.org"